scripts/drbd

```diff
@@ -166,9 +172,6 @@ handle_linstor_loopback()
 			stop)  losetup "$dev" 2>/dev/null && losetup -d "$dev";;
 		esac
 	done < "$loop_mapping"
-
-	# at least we tried.
-	return 0
 }
```

Comment:
let's keep it. even if setup fails we "at least tried", and I'd like to keep the `0` return value